backbone.js

```diff
@@ -1408,6 +1408,31 @@
     return Backbone.$.ajax.apply(Backbone.$, arguments);
   };
 
+  // A psuedo Promise implementation used to ensure asynchronous methods
+  // return thenables.
+  // Override this if you'd like to use a different ES6 library.
+  Backbone.Promise = function() {
```

Comment:
@megawac but does it fail for our use case here? Backbone and Underscore tend to opt for the ponyfill approach over pollyfills, encapsulating the desired behavior in functions instead of shimming support globally.

If we force a Promise library (or a Promise global), it'll be confusing to developers why this fails in older browsers but works in newer ones. On the other hand if we set the precedent here that we're ok with making users pollyfill older behaviors, we can start to [follow the lead of React](https://facebook.github.io/react/docs/working-with-the-browser.html#browser-support-and-polyfills) in requiring developers to pollyfill native methods (map, forEach, etc).